.golangci.yaml

```diff
@@ -14,6 +14,7 @@ linters:
   enable-all: true
   disable:
     # Disabled because not relevant
+    - sloglint
```

Comment:
disabled cuz it's failing for no reason only in CI. Locally it's ok. I suppose it's related to some bug in newer version of golangci lint